components/gorgias_oauth/actions/create-customer/create-customer.mjs

```diff
@@ -3,11 +3,12 @@ import overrideApp from "../../common/override-app.mjs";
 
 overrideApp(base);
 
+// same version as base
+// eslint-disable-next-line pipedream/required-properties-version
 export default {
   ...base,
   key: "gorgias_oauth-create-customer",
   name: "Create Customer",
   description: "Create a new customer. [See the docs](https://developers.gorgias.com/reference/post_api-customers)",
-  version: "0.0.1",
```

Comment:
Sorry. I don't get it exactly even with the comment. Why are you removing the version here and in other files?

Reply:
This action has the same behaviour defined in `gorgias-create-customer`. If the code from `gorgias-create-customer` is changed and published, we have to rev the version of `gorgias_oauth-create-customer` and publish it too.

To avoid forgetting to do this, removing `version` will make this action always have the same version as the `base` and be published together.